Parsedown.php

```diff
@@ -506,6 +506,11 @@ protected function blockHeader($Line)
             }
 
             $text = trim($Line['text'], '#');
+
+            if (!isset($text[0]) or $text[0] !== ' ') {
```

Comment:
If this were to be merged, I think in order to give some recourse to users that do not want the space requirement, it would be a good to add a new setter which can turn this behaviour off.

Reply:
Easy enough to add a setting to maintain backward compat.